op-program/host/cmd/main_test.go

```diff
@@ -256,6 +257,11 @@ func TestL2Head(t *testing.T) {
 		delete(req, "--l2.head")
 		delete(req, "--l2.outputroot")
 		args := append(toArgList(req), "--l2.agreed-prestate", "0x1234")
+		// TODO: Remove the --depset.config flag once there's a depset defined for sepolia.
```

Comment:
nit: Can we add an issue reference for this and the other TODOs in this file please? It's the kind of thing we'll easily forget otherwise. Could be an issue just for this if need be or for adding the sepolia depset to superchain registry.

Reply:
totally. Fixed.